src/templates/homepage/HeroSection.jsx

```diff
@@ -24,53 +24,59 @@ const HeroDropdownElem = ({ title }) => (
   </div>
 )
 
-const HeroDropdown = ({ title, options, isActive, toggleDropdown }) => (
-  <div
-    className={`bp-dropdown margin--top--sm ${isActive ? "is-active" : null}`}
-  >
-    <div className="bp-dropdown-trigger">
-      <a
-        className="bp-button bp-dropdown-button hero-dropdown is-centered"
-        role="button"
-        tabIndex="0"
-        aria-haspopup="true"
-        aria-controls="hero-dropdown-menu"
-        onClick={toggleDropdown}
-        onKeyDown={toggleDropdown}
-      >
-        <span>
-          <b>
-            <p>{title}</p>
-          </b>
-        </span>
-        <span className="icon is-small">
-          <i
-            className="sgds-icon sgds-icon-chevron-down is-size-4"
-            aria-hidden="true"
-          />
-        </span>
-      </a>
-    </div>
+const HeroDropdown = ({ title, options }) => {
+  const [dropdownIsActive, setDropdownIsActive] = useState(false)
+  const toggleDropdown = () => setDropdownIsActive((prevState) => !prevState)
```

Comment:
if this pattern comes out often we can consider writing a custom hook

Reply:
Not tackling this in within this refactor as this pattern isn't that common in our CMS.